### DISCUSSION TITLE
Make `storage.postgres.password` not required, for users of peer based authentication?

### DISCUSSION THREAD

**Discussion by poperigby** [authelia/authelia#7646] [Ideas]
If you're using peer based authentication, using a password isn't required. Could the option be either set as not required, or only set as not required if a Unix socket is being used?

**Comment by james-d-elliott** [comment]
Yeah this may be a decent idea. Do you happen to know if MySQL has similar functionality?

**Reply by poperigby** [reply]
Sorry for the delay. Yeah, it looks like both MySQL and MariaDB support something similar:

* https://dev.mysql.com/doc/refman/8.4/en/socket-pluggable-authentication.html
* https://mariadb.com/kb/en/pluggable-authentication-overview/#unix_socket

**Comment by james-d-elliott** [comment]
This was added, think in 4.39.
